tests/test_utilities.py

```diff
@@ -8,9 +9,17 @@
 from Crypto.PublicKey import RSA
 
 from smile_id_core import Signature, Utilities
+from smile_id_core.Utilities import get_version
 from tests.stub_mixin import TestCaseWithStubs
 
 
+def test_get_version():
+    """Test that a well formatted version string is returned."""
+    assert isinstance(get_version(), str)
+    # tests that the version is in the form of a semver string.
+    assert re.match(r"^\d+\.\d+\.\d+$", get_version())
+
```

Comment:
Can you also assert that `__version__` exists?

Reply:
test for __version__ has been added to `setup_config_test.py`